server/server/database/test_data/semester_dummy.sql

```diff
@@ -3,7 +3,7 @@ VALUES
     ('2018-19 Fall / Spring', 'SE', '2018-08-27', '2019-05-08'),
     ('2019-19 Spring / Summer', 'SE', '2019-01-13', '2019-08-07'),
     ('2019-20 Fall / Spring', 'SE', '2024-05-27', '2024-08-08'),
-    ('2024-24 Spring / Summer', 'SE', '2024-01-13', '2024-08-07'),
+    (CONCAT(STRFTIME('%Y', DATE(DATE('now'), '-2 MONTHS')), '-', STRFTIME('%yy', DATE(DATE('now'), '+3.5 MONTHS')),  ' Spring / Summer'), 'SE', DATE(DATE('now'), '-2 MONTHS'), DATE(DATE('now'), '+3.5 MONTHS')),
```

Comment:
I went a little overboard here, haha. At a minimum, the change can be like this. All of the other stuff just makes sure the years in the semester name are correct. I don't mind keeping it simple if that's easier to understand/replicate.

```suggestion
    ('2024-24 Spring / Summer', 'SE', DATE(DATE('now'), '-2 MONTHS'), DATE(DATE('now'), '+3.5 MONTHS')),
```